CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributor Guidelines
+
+We value and encourage contribution from the community. To reduce
+friction in this process, we have collected some best-practices for
+contributors:
+
+* API style. We have chosen to split the API of the modules in our 
+  benchmarking package in the following way:
+  `exp_obj = generate_X_experiment( parameters)`
+  `results_obj = acquire_data_X(exp_obj)`
```

Comment:
Sorry I missed this in the qubit specs refactor: the majority of existing modules use acquire_X_data, similar to acquire_X_experiment. Shall we change it here and make a note for the specs module?

Reply:
Good point. I'll change the specs module.

Reply:
this is now in a branch so I will resolve this commet